README.md

```diff
@@ -30,7 +30,7 @@ Insights Remediations supports the following types of remediations:
 
 ## Documentation
 
-* [API Documentation](https://remediations-ci.5a9f.insights-dev.openshiftapps.com)
+* [API Documentation](https://developers.redhat.com/api-catalog/api/remediations)
```

Comment:
I find the new API doc format useless.  Lets use https://console.redhat.com/docs/api/remediations instead.

Reply:
Done!